test/api_wrong_credentials_search_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:openfoodfacts/model/parameter/SearchTerms.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:openfoodfacts/utils/QueryType.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  test('Test for FormatException when searching with wrong credentials',
```

Comment:
You should also run the same test
* with the TEST user, in order to check that things will work fine when they are supposed to.
* with a decent user but a wrong password (I assume that the user `sdfsdf` does not exists)